pkg/users/local.js

```diff
@@ -95,14 +121,23 @@ function AccountsPage() {
                 groups={groupsExtraInfo || []}
                 isGroupsExpanded={isGroupsExpanded}
                 setIsGroupsExpanded={setIsGroupsExpanded}
+                min_gid={min_gid}
+                max_gid={max_gid}
             />
         );
     } else if (path.length === 1) {
         return (
             <AccountDetails accounts={accountsInfo} groups={groupsExtraInfo} shadow={shadow || []}
                             current_user={current_user_info && current_user_info.name} user={path[0]} />
         );
-    }
+    } else if (path.length === 2 && path[0] === "group" && groupsExtraInfo.length > 0 && accountsInfo.length > 0) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230221-091922-eab3b34f-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)